package.json

```diff
@@ -13,14 +13,12 @@
     "node": ">=6"
   },
   "bin": "index.js",
-  "files": [
-    "index.js",
-    "src"
-  ],
+  "files": ["index.js", "src"],
   "scripts": {
     "precommit": "node index.js",
     "cz": "git-cz",
-    "lint": "eslint .",
+    "lint:base": "eslint --rule 'prettier/prettier: 2'",
```

Comment:
Could you change this to `"lint:base": "eslint --rule \"prettier/prettier: 2\""`? Single quotes don't work correctly for me on Windows.